requirements-types.txt

```diff
@@ -1,9 +1,10 @@
 --requirement requirements.txt
+--requirement reqs/requirements-dev-lite.txt
```

Comment:
`dev-lite` was a transitive dependency of this CI step before. Now I'm making it explicit.